box_set1:
  box1: [0.115, 0.100, 0.075]
  box2: [0.160, 0.125, 0.075]
  box3: [0.100, 0.075, 0.075]
  box4: [0.195, 0.160, 0.075]
  box5: [0.150, 0.100, 0.075]
  box6: [0.150, 0.100, 0.075]
  box7: [0.160, 0.125, 0.075]
  box8: [0.150, 0.150, 0.150]
  box9: [0.100, 0.100, 0.075]
  box10: [0.195, 0.160, 0.150]
  box11: [0.135, 0.075, 0.075]
  box12: [0.160, 0.125, 0.075]
  box13: [0.150, 0.100, 0.075]
  box14: [0.195, 0.160, 0.150]
  box15: [0.195, 0.160, 0.075]
  box16: [0.150, 0.150, 0.150]
  box17: [0.135, 0.075, 0.075]
  box18: [0.150, 0.150, 0.150]
  box19: [0.135, 0.075, 0.075]
  box20: [0.115, 0.100, 0.075]
  
box_set2:
  box1: [0.160, 0.125, 0.075]
  box2: [0.195, 0.160, 0.150]
  box3: [0.115, 0.100, 0.075]
  box4: [0.100, 0.100, 0.075]
  box5: [0.195, 0.160, 0.150]
  box6: [0.135, 0.075, 0.075]
  box7: [0.150, 0.100, 0.075]
  box8: [0.075, 0.075, 0.075]
  box9: [0.100, 0.100, 0.075]
  box10: [0.195, 0.160, 0.150]
  box11: [0.135, 0.075, 0.075]
  box12: [0.160, 0.125, 0.075]
  box13: [0.150, 0.100, 0.075]
  box14: [0.195, 0.160, 0.150]
  box15: [0.195, 0.160, 0.075]
  box16: [0.150, 0.150, 0.150]
  box17: [0.135, 0.075, 0.075]
  box18: [0.150, 0.150, 0.150]
  box19: [0.135, 0.075, 0.075]
  box20: [0.115, 0.100, 0.075]

